.github/workflows/release.yml

```
@@ -90,37 +109,43 @@ jobs:
 
           - name: Compress xz
             if: matrix.os != 'windows'
-            run: xz -T0 -9 sonicweb-*-*
+            run: |
+                  tar -cf - sonicweb-${{ matrix.os }}-${{ matrix.architecture }} \
+                            third_party_licenses.tar.xz                          \
+                | xz -T0 -9 sonicweb-${{ matrix.os }}-${{ matrix.architecture }}.tar.xz
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected comment or line break

(syntax-check)